packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -237,13 +240,15 @@ export class RecentsWidget extends Component {
 
     // Recents widget is ready once we have some spaces or a search causing spaces to be empty
     const isFiltered = keywordFilter && keywordFilter.length;
-    const isReady = spacesListArray && (spacesListArray.length || isFiltered);
+    const isReady = (widgetStatus.hasFetchedInitialSpaces || isFiltered);
 
     const isLoadingMore = props.extendedLoad
       ? !widgetStatus.hasFetchedAllSpaces : !widgetStatus.hasFetchedInitialSpaces;
 
     // Display the header if any of these options are true, otherwise, hide header.
     const showHeader = enableSpaceListFilter || enableAddButton || enableUserProfile;
+    const hasSpaces = widgetStatus.hasFetchedInitialSpaces && spacesListArray.length > 0;
+    const emptyMessage = !hasSpaces && enableAddButton ? 'Create a space using the plus button next to the search bar above.' : 'Create spaces in Webex Teams to see them here.';
```

Comment:
Can these messages be added to the `messages.js` for eventual internationalization?

Reply:
yepp, will do.